installing/installing_oci/installing-oci-assisted-installer.adoc

```diff
@@ -48,8 +51,7 @@ include::modules/installing-troubleshooting-assisted-installer-oci.adoc[leveloff
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc#using-the-assisted-installer_installing-on-prem-assisted[Installing an on-premise cluster using the Assisted Installer]
-* link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index[Assisted Installer for {product-title}]
-//* link:https://access.redhat.com/node/7038262[Using the Agent-based Installer to install a cluster on OCI]
-* link:https://docs.oracle.com/en-us/iaas/Content/ResourceManager/Concepts/resourcemanager.htm#ways[Ways to access Resource Manager]
-* link:https://docs.oracle.com/en-us/iaas/Content/ResourceManager/Tasks/create-stack.htm#top[Creating a stack] in the Oracle documentation.
+* xref:../../installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc#using-the-assisted-installer_installing-on-prem-assisted[Installing an on-premise cluster using the {ai-full}]
+* link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index[{ai-full} for {product-title}]
```

Comment:
This link needs to be updated.

Reply:
Thanks. For a follow-up PR. I need to address merge conflicts on 4.14-4.16